.github/workflows/link-check.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           url: "https://www.featureform.com/"
           # optional:
           honorRobotExclusions: false
-          ignorePatterns: "*linkedin*,*autodesk*,*crunchbase*" # Linkedin returns 999 error to web scrapers
+          ignorePatterns: "*linkedin*,*autodesk*,*crunchbase*,*springboard*" # Linkedin returns 999 error to web scrapers
```

Comment:
@MMBazel thoughts on adding springboard to the ignore list?

Reply:
actually, since the job is flaky (and as you pointed out the link works), I'll revert this change.